<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeySignatureDialog</class>
 <widget class="QDialog" name="KeySignatureDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Key Signature</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="visibleLabel">
       <property name="text">
        <string>Visible:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="visibilityCheckBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="keyTypeLabel">
       <property name="text">
        <string>Key Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="majorKeyButton">
         <property name="text">
          <string>Major</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="minorKeyButton">
         <property name="text">
          <string>Minor</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="keyLabel">
       <property name="text">
        <string>Key:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="keysComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
</ui>
